build/Targets/Roslyn.Toolsets.Xunit.targets

```diff
@@ -18,4 +18,22 @@
     <StartArguments Condition="'$(StartArguments)' == ''">$(XUnitArguments)</StartArguments>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <PrepareForBuildDependsOn>$(PrepareForBuildDependsOn);AddDefaultTestAppConfig</PrepareForBuildDependsOn>
+  </PropertyGroup>
+
+  <!-- Add a default test app.config, if the project doesn't already have one-->
+  <Target Name="AddDefaultTestAppConfig">
+    <PropertyGroup>
+      <_AlreadyHasAppConfig Condition="'%(None.Filename)%(None.Extension)' == 'app.config'">true</_AlreadyHasAppConfig>
```

Comment:
None is what it looks at first, than content. I excluded content at the moment, but I'll add in a followup.